scripts/cypress.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -eux
+
+echo "Starting backend..."
+./plugin-backend --loglevel info --config ./config/config.yaml &
+backend=$!
+
+echo "Running tests..."
+cd web
+npm run cypress:run
+cypress=$?
+
+kill $backend
+wait $backend
```

Comment:
do you kill before you wait?

Reply:
never mind, I see you're just waiting process to terminate after killing.

Reply:
yeah kind of disapointing reading these lines but it's waiting for process to end indeed :+1: